scripts/generators/fetch.js

```diff
@@ -40,35 +57,35 @@ async function fetchAdditionalText(container) {
     throw new Error(`container ${container.id} is empty`);
   }
   const nbBaseText = container.texteBaseId.length;
-  const textedeBase = container.sections
-    .slice(0, nbBaseText)
-    .filter(isValidSection);
-  const additionnalSections = container.sections
-    .slice(nbBaseText)
-    .filter(isValidSection);
-
-  const pAdditionnalSections = additionnalSections.map(async (mainSection) => {
-    const pSections = mainSection.sections.filter(isValidSection).map((text) =>
+  const textedeBase = container.sections.slice(0, nbBaseText).filter(isValidSection);
+  const additionnalSections = container.sections.slice(nbBaseText).filter(isValidSection);
+
+  const pAdditionnalSections = additionnalSections.map(async mainSection => {
+    const pSections = mainSection.sections.filter(isValidSection).map(text =>
       queue.add(() => {
         return retry(() => getKaliText(text.id), { retries: 10 });
-      })
+      }),
     );
     mainSection.sections = await Promise.all(pSections);
-    mainSection.sections.forEach((section) => {
+    mainSection.sections.forEach(section => {
       section.etat = section.jurisState;
     });
+
     return mainSection;
   });
+
   // Ensure Additionnal Text Section is not empty
   const sectionsWithText = (await Promise.all(pAdditionnalSections)).filter(
-    ({ sections }) => sections.length > 0
+    ({ sections }) => sections.length > 0,
   );
-  container.sections = [...textedeBase, ...sectionsWithText];
+  container.sections = [...textedeBase, ...sectionsWithText].sort(sortText);
```

Comment:
beware of latest changes here : https://github.com/SocialGouv/kali-data/commit/cc021816d69766368bbea47e785308d082a3c965